onnx_tf/handlers/backend/upsample.py

```diff
@@ -34,7 +34,7 @@ def version_7(cls, node, **kwargs):
     new_weight = np.floor(x_shape[3] * scales[3])
 
     mode = attrs.get("mode", "nearest")
-    if mode.lower() == "bilinear":
+    if mode.lower() == "bilinear" or "linear":
```

Comment:
This is not valid syntax. It would always return true, and thus always use bilinear interpolation.